content/integrations/awslambda.md

```diff
@@ -21,27 +21,93 @@ If you haven't already, set up the [Amazon Web Services integration first](/inte
 In the Amazon Web Services integration tile, ensure that Lambda is checked under metric collection.
 
 To send custom metrics to Datadog, you must print a log line from your Lambda, using the following format:
-`MONITORING|unix_epoch_timestamp|value|metric_type|my.metric.name|#tag1:value,tag2`
 
-Please ensure the `unix_epoch_timestamp` is in seconds (not milliseconds).
+~~~
+MONITORING|unix_epoch_timestamp|value|metric_type|metric.name|#tag1:value,tag2
+~~~
+
+## Notes on each section
+
+### MONITORING
+* This is used to find the log statement within Cloudwatch Logs and pull it into Datadog
+
+### unix_epoch_timestamp
+* Please ensure the `unix_epoch_timestamp` is in seconds (not milliseconds)
+
+### value
+* The value of the metric **must** be a number.
+* For metric_type `check` the options are, `'0': OK, '1': WARNING, '2': CRITICAL, '3': UNKNOWN`
+
+### metric_type
+* Supported metric types are `count, gauge, histogram, and check`
```

Comment:
The list of types are all in one code block (they shouldn't be)

This can be simple:

"metric_type: `count`, `gauge`, `histogram`, or `check`"